doc/nrf/device_guides/nrf54h.rst

```diff
@@ -0,0 +1,35 @@
+.. _ug_nrf54h:
+
+Working with nRF54H Series
+##########################
+
+.. note::
+
+  All software for the nRF54H20 SoC is experimental and hardware availability is restricted to the participants in the limited sampling program.
+
+Zephyr and the |NCS| provide support and contain board definitions for developing on the following nRF54H Series device:
+
+.. list-table::
+   :header-rows: 1
+
+   * - DK
+     - PCA number
+     - Build target
+     - Documentation
+   * - :ref:`zephyr:nrf54h20dk_nrf54h20`
+     - PCA
+     - | ``nrf54h20dk_nrf54h20_cpuapp``
+       | ``nrf54h20dk_nrf54h20_cpurad``
+       | ``nrf54h20dk_nrf54h20_cpuppr``
+     - --
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Subpages:
+
+   working_with_nrf/nrf54h/ug_nrf54h20_gs
```

Comment:
Since now all the other device GSGs are moved to the getting started guides section, we could add a bullet point for nRF54h20_gs as well in the note admonitions, like we have for the nRF54l15 - https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gsg_guides.html